kong/globalpatches.lua

```diff
@@ -15,15 +17,16 @@ return function(options)
   local meta = require "kong.meta"
 
 
-  local cjson = require("cjson.safe")
-  cjson.encode_sparse_array(nil, nil, 2^15)
+  local cjson_safe = require("cjson.safe")
```

Comment:
Could we replace it with `kong.tools.cjson`?

Reply:
I doubt you can, since this is in globalpatches, which gets loaded very early on.

Reply:
You can't simply replace it, because currently, our global patch doesn't set `cjson.decode_array_with_array_mt(true)`, whereas `kong.tools.cjson` does. https://github.com/Kong/kong/blob/b7a83612f4cf87404e758b42087ff1623a916eb9/kong/tools/cjson.lua#L4

Reply:
Got. The code makes me a bit confusing.